update_python_spec.lisp

```diff
@@ -1,13 +1,23 @@
 #!/usr/bin/env sbcl --script
 (load "~/.sbclrc")
 (ql:quickload :rpcq)
+(ql:quickload :split-sequence)
 (rpcq::clear-messages)
 
+(defun -to_ (f)
```

Comment:
Maybe I'm tired and that's why I can't write a simple solution for search and replace, or find one within CL.

Reply:
We have a variant of this function elsewhere in rpcq already: https://github.com/rigetti/rpcq/blob/master/src/utilities.lisp#L8 .

Reply:
@notmgsk 
```
CL-USER> (substitute #\_ #\- "Hello-world_there")
"Hello_world_there"
```